packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,126 @@
+| `max-date`                 | string                                                  | -                 | The maximum allowable date                                                                                                        |
+| `show-link`                | boolean                                                 | `true`            | If false, link at link slot is hidden                                                                                             |
+| `highlight-current-month`  | boolean                                                 | -                 | if true, the month picker should highlight the current month                                                                      |
+| `highlight-selected-month` | boolean                                                 | -                 | If true, the month picker should highlight the selected month                                                                     |
+| `today`                    | string                                                  | -                 | value of today                                                                                                                    |
+| `selected-dates`           | string[]                                                | -                 | selected dates                                                                                                                    |
+
+### Outputs
+
+- [selectedDates: string] - the selected dates from the calendar
+
+### Events
+
+- `change`: Emits when the selected date(s) is changed
+- `date-select`: Emits when user selects a date(s) when no date(s) is currently selected
+- `view-change`: Emits when view changes (e.g. from date picker to month picker) via user interaction
+
+### Slots
+
+- `start` - content before the calendar
+- `end` - content after the calendar
+- `link` - link at the bottom right of the base of the calendar (e.g. "Go to today")
+- `navicon-left` - left icon on header to navigate backwards for months, years, or range of years
+- `navicon-right` - right icon on header to navigate forwards for months, years, or range of years
+
+Note that the `default` slot does not contain any customizable content.
+
+### CSS Variables
+
+None
+
+## Accessibility
+
+- [ ] Find the matching component through [WCAG's patterns](https://www.w3.org/WAI/ARIA/apg/patterns/)
+  - [Date Picker Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/)
+- [ ] Are there any accessibility elements unique to this component?
+- [ ] List ARIA attributes: `role, aria-labelledby, aria-label, aria-pressed, aria-selected, aria-live`
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
```

Comment:
Keyboard behavior is a good topic to think through before getting into knee deep into dev work on this.